app/views/admin/invoices/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="invoices-header">
+  <h2> Invoices </h2>
+</div>
```

Comment:
Maybe incorporate the partial header that Daniel made?

Reply:
I tried to use that partial but it does not seem to work. when i do it i get an error that says 
```
Missing partial admin/invoices/_admin_header, application/_admin_header with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :jbuilder]}.
```

Reply:
As of now the _admin_header and the _new_edit partial work :)

Reply:
I added this partial now